### DISCUSSION TITLE
Open tabs, collapsible when there are errors on the page

### DISCUSSION THREAD

**Discussion by matthijs166** [payloadcms/payload#881] [Roadmap]
# Bug Report

I'm working on a large project and have a lot of blocks on the page.
Thing is when I skip a tab or a collapsible and get a validation error its a lot of clicking to find the errored field

## Expected Behavior

When there is an error I expect that I can find the fields with the error easy

## Current Behavior

When there is an error before saving a collection I need to search where the field that has the error.

## Possible Solution

Open the first tab or collapsible where the error is and scroll to it

## Detailed Description

Payload 1.0.12
Chrome

**Comment by jmikrut** [comment]
Hey @matthijs166 — thanks for bringing this up! We're actually currently working on a feature to make it so that nested errors (which might not be immediately visible on the page) are still visible through the "chain" of fields that they reside in.

I'm going to convert this to a Discussion as it's more of a feature request, but we are actively working on improving this UX and will have some updates for the way this is all handled shortly!

**Reply by matthijs166** [reply]
Im very excited to see the new feature! Thanks

**Reply by emilkarl** [reply]
Awesome!
